docker/Dockerfile

```diff
@@ -66,8 +70,11 @@ RUN cp -a /usr/bin/sdk solana
 RUN ./pyth-client/scripts/patch-solana.sh
 
 # Build and test the oracle program.
-RUN cd pyth-client && ./scripts/build-bpf.sh .
+RUN cd pyth-client && ./scripts/build-bpf.sh . "test"
```

Comment:
it's confusing to run the tests on a different version of the code than the one that gets released. What happens if the tests pass on the version with the feature flag enabled, but fail on the other version? 

I think you're doing this to test your code, but it would be better to find a different way to run the tests.

Reply:
Yeah the reason for this is that the resizing feature is disabled in production

Reply:
fair point, I recall that you and I and Guillermo talked about this, and we said we wanted to have a different "feature" for resizing accounts.
Should I run the tests twice? that would take ~40 minutes. or should maybe run them on the production version, then redeploy the "test" version and just test the resize and publish instruction?

Reply:
Could we run them twice, but configure the GitHub Actions workflow to do it in parallel?